app/ide-desktop/lib/dashboard/src/pages/subscribe/Subscribe.tsx

```diff
@@ -16,6 +16,7 @@ import * as backendProvider from '#/providers/BackendProvider'
 import * as textProvider from '#/providers/TextProvider'
 
 import Modal from '#/components/Modal'
+import UnstyledButton from '#/components/styled/UnstyledButton'
```

Comment:
Not sure it's a good idea to put `UnstyledButton` under the `styled` folder

Reply:
true... the long-term intention was to slowly turn all components into `styled` components and then move `styled/*` into the root. but to be fair it doesn't hurt to *not* put this file in the wrong place...